File: ERP_Anass_app/src/app/models/Finance/Account.ts

export enum AccountType {
    Asset = 1,
    Liability = 2,
    Equity = 3,
    Income = 4,
    Expense = 5
}

export interface Account {
    idAccount?: number;
    accountCode?: string;
    accountName?: string;
    type?: AccountType;
    isAcitve?: boolean;
}

File: ERP_SERVER/ERP_Anass_backend/Models/Account.cs
using System.ComponentModel.DataAnnotations;

namespace ERP_Anass_backend.Models
{
    public enum AccountType
    {
        Asset,
        Liability,
        Equity,
        Income,
        Expense
    }

    public class Account
    {
        [Key]
        public int idAccount { get; set; }
        public string AccountCode { get; set; }
        public string AccountName { get; set; }
        public AccountType Type { get; set; }
        public Boolean IsAcitve { get; set; } = true;
    }
}
